base/files/file_enumerator_unittest.cc

```diff
@@ -419,12 +419,15 @@ TEST(FileEnumerator, InvalidDirectory) {
   FilePath path = enumerator.Next();
   EXPECT_TRUE(path.empty());
 
-  // Slightly different outcomes between Windows and POSIX.
-#if BUILDFLAG(IS_WIN)
-  EXPECT_EQ(File::Error::FILE_ERROR_FAILED, enumerator.GetError());
-#else
-  EXPECT_EQ(File::Error::FILE_ERROR_NOT_A_DIRECTORY, enumerator.GetError());
-#endif
+// Different behaviour between directory APIs and posix mappings
+// specifically in how we set error codes
+//
```

Comment:
Did you want to leave this commented out here ? If this isn't resolvable in this PR, maybe lets create a small follow-up bug to clean this up and link here as a TODO(b/1234)